src/Wallabag/CoreBundle/Entity/Tag.php

```diff
@@ -13,7 +13,10 @@
  * Tag.
  *
  * @XmlRoot("tag")
- * @ORM\Table(name="`tag`")
+ * @ORM\Table(
+ *     name="`tag`",
+ *     options={"collate"="utf8mb4_bin", "charset"="utf8mb4"},
```

Comment:
Don't you need to update the migration accordingly?
How is it handled by PG & SQLite as they shouldn't know what utf8mb4 is?